build/index.html

```diff
@@ -1,12 +1,12 @@
-<!DOCTYPE html>
```

Comment:
The build folder is somehow making it into the commit despite it being in the gitignore. This shouldn't be part of the commit.